src/Stackify/Log/Monolog/Handler.php

```diff
@@ -28,12 +28,24 @@ public function __construct($appName, $environmentName = null, TransportInterfac
         $this->transport = $transport;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function handle(array $record)
     {
+        if (!$this->isHandling($record)) {
```

Comment:
Can you explain what this is doing?

Reply:
- The minimum logging level at which this handler will be triggered